docs/fides/mkdocs.yml

```diff
@@ -6,6 +6,7 @@ nav:
       - Overview: getting_started/overview.md
       - Fidesctl in Docker: getting_started/docker.md
       - Fidesctl Manually (without Docker): getting_started/manual.md
+  - Installation: installation.md
```

Comment:
I realize the "Getting Started" docs are for development, but this title would give me pause as a new user. Do I want to install fidesctl or get started with it? I think it's worth renaming the getting started guide to specify.

Reply:
this basically seeds the ground for #200 , which consolidates these sections